tools/tests/test_cases/test_gemm.cpp

```diff
@@ -58,6 +58,19 @@ TEST_F(DnnlPluginNext_GEMM, Resnet50_Fp16)
     run();
 }
 
+TEST_F(DnnlPluginNext_GEMM, LargeSize_Fp32)
+{
+    if (!g_test_config.run_dml)
+    {
+        GTEST_SKIP() << "Enabled only for DML path. ToDo: add support for non-DML path.";
+    }
+    params_.shape_a = TensorShape(1,1,144,4096);
+    params_.shape_b = TensorShape(1,1,4096,1024);
+    params_.b_managed = true;
```

Comment:
I see you've added some code regarding c_managed, but it's not used here - is that intended?

Reply:
yes, this is not used for this case. However, I have added other cases in local debug and saw c_managed is not correctly passed to the setting. That is the reason I also added some change for c_managed.